Update translations for Portuguese in codelists for schema 19139

julianofinck opened the pull request:
In the Portuguese codelist for the schema 19139, few words had typos and others were in Spanish.
  
# Checklist

- [x] I have read the [contribution guidelines](https://github.com/geonetwork/core-geonetwork/blob/main/CONTRIBUTING.md)
- [x] *Pull request* provided for `main` branch, backports managed with label
- [x] *Good housekeeping* of code, cleaning up comments, tests, and documentation
- [x] *Clean commit history* broken into understandable chucks, avoiding big commits with hundreds of files, cautious of reformatting and whitespace changes
- [x] *Clean commit message*s, longer verbose messages are encouraged
- [ ] *API Changes* are identified in commit messages
- [ ] *Testing* provided for features or enhancements using [automatic tests](https://github.com/geonetwork/core-geonetwork/blob/main/software_development/TESTING.md)
- [ ] *User documentation* provided for new features or enhancements in [manual](https://github.com/geonetwork/core-geonetwork/tree/main/docs/manual)
- [ ] *Build documentation* provided for development instructions in `README.md` files
- [ ] *Library management* using `pom.xml` dependency management. Update build documentation with intended library use and library tutorials or documentation

julianofinck commented:
If this PR gets accepted, does it also gets applied to the 3.10.X branch that had the last commit 2 years ago, or we do not perform backward adjustments?

josegar74 commented:
@julianofinck The encoding of the file you committed in the pull request doesn't look fine:

https://github.com/geonetwork/core-geonetwork/pull/8521/files#diff-06541c6e9fe74ce5a701c20deac7c4577b54e9f7e1d7d187f2a4b138429bb1c9R89

Compare to the original file:

https://github.com/geonetwork/core-geonetwork/blob/908338e6c262a5071aa5b5a6f57c9591c3467b33/schemas/iso19139/src/main/plugin/iso19139/loc/por/codelists.xml#L89

julianofinck commented:
@josegar74, thank you very much for your feedback!

I believe you meant the encoding "ISO-8859-1" was not fine, although the ISO19139 Portuguese's codelist.xml was in this encoding from the start: [codelist.xml](https://github.com/geonetwork/core-geonetwork/blob/main/schemas/iso19139/src/main/plugin/iso19139/loc/por/codelists.xml)

After inspection, I realized all other languages had this file in "UTF-8". I thus re-saved it with the encoding "UTF-8" (and changed the XML header as well).

Presumably the encoding is fine right now. If not, do not hesitate to tell me.
PS: I had to force-push because had pushed many commits before that looked ugly. I squashed them in a more logical way.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/geonetwork/core-geonetwork?pullRequest=8521) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/geonetwork/core-geonetwork?pullRequest=8521) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/geonetwork/core-geonetwork?pullRequest=8521) it.</sub>

julianofinck commented:
The link to sign the CLA is returning 500 with message:
_Error - There is no CLA to sign for geonetwork/core-geonetwork
(could not find linked item for owner geonetwork and repo core-geonetwork)_

Must it be issued again?

josegar74 commented:
@julianofinck I don't know about this new CLA stuff.

I'll check the updated changes this week and merge them.

josegar74 commented:
@julianofinck the pull request works fine. I found a problem in the `codelist.xml` not related to your changes, but it would be nice to fix it.

In the add online resources the function list doesn't show values in Portuguese:

![online-resource-function](https://github.com/user-attachments/assets/8793e8c3-c0ab-495a-983d-71acfd3057ed)

The following line:

```xml
<codelist name="gmd:CI_OnLineFunctionCode">
```

Needs to be updated to 

```xml
<codelist name="gmd:CI_OnLineFunctionCode" alias="function">
```

Can you update the pull request with this change?

julianofinck commented:
Hi, @josegar74. The requested change is done.
I thought it would take way longer so waited for a not busy sunday.
